uniffi_bindgen/src/bindings/python/templates/UInt8Helper.py

```diff
@@ -1,4 +1,10 @@
 class FfiConverterUInt8(FfiConverterPrimitive):
+    @classmethod
+    def lower(cls, value):
+        if not 0 <= value < 2**8:
+            raise ValueError("u8 requires {} <= value < {}".format(0, 2**8))
```

Comment:
I'd love to reduce the code duplication here. What about this?
  - Define `class FfiConverterInt(FfiConvererPrimitive)`
  - `FfiConverterInt` overrides lower and adds the bounds check, but doesn't define the upper/lower bounds.
  - Each integer type inherits from `FfiConverterInt` and defines the bounds.

Reply:
That would work, it'd come with a (tiny) performance penalty, but I guess that's okay.

Reply:
Done.